ciongke/cmd/hook/main.go

```diff
@@ -53,6 +53,11 @@ var defaultJenkinsJobs = map[string][]JenkinsJob{
 			Context:   "Jenkins GKE smoke e2e",
 		},
 		{
+			Name:    "kubernetes-pull-gci-build-test-e2e-gce",
+			Trigger: regexp.MustCompile(`@k8s-bot (gci )?(gce )?(e2e )?test this`),
```

Comment:
Oh, sorry I missed this before but if `@k8s-bot test this` triggers it then it will run for every PR before merging, which we don't want. Could you make the gci bit required? If we start running it for every PR then we will switch back to this.